docs/Cloud_integrations/Device_Integration_Center/overview.md

```diff
@@ -0,0 +1,88 @@
+# Data Integration Center
+
+Data Integration Center is our feature that allows you to forward device related data to your systems, databases,
+message brokers etc. There are two kind of events:
+
+* Lifecycle - these are related to device life cycle events like creation, update, deletion
+* Telemetry - these are related to telemetry gathered by device like temperature measurement
+
+![overview](./images/data-integration-center-overview.webp)
+
+## Templating
+
+Templating feature allow you to define how your payload (or headers) looks like. This enables you to integrate with
+almost all available services, gathering your data where you desire.
+
+![wizard-form-body](./images/wizard-form-body.webp){ width=50% }
+
+You are free to modify body template as long as it proper JSON. If you want different content type you can use `Text`
```

Comment:
You can modify a body template as long as it is proper JSON. If you want a different content type you can use `Text` that have no validation at all.